industrial_ci/src/tests/clang_format_check.sh

```diff
@@ -19,30 +19,32 @@ function run_clang_format_check() {
   local err=0
   local path=$TARGET_REPO_PATH
 
-  ici_require_run_in_docker # this script must be run in docker
-  
+  DOCKER_IMAGE="$DOCKER_BASE_IMAGE" ici_require_run_in_docker # this script must be run in docker
```

Comment:
This depends on https://github.com/ros-industrial/industrial_ci/pull/329?

The image would be quite arbitrary. Could we not give some default Docker image that should always work with clang?

Reply:
It does not depend on the other pull request.
Hard-coding would be another option, but I am not sure if this is a good idea.

If you want to hard-code, then https://github.com/ros-industrial/industrial_ci/pull/329 is needed.
If we stick to ROS_DISTRO, then the clang version probably matches the one that is installed in the user's environment